src/apps/frontend/constants/country-code.ts

```diff
@@ -0,0 +1,77 @@
+interface CountryCode {
+  dial_code: string;
```

Comment:
- `_` is not a proper JS naming convention, please use `camelCase`
- Properties are also not ordered alphabetically

Reply:
The name of the file is not clear, the file contains more than country code.